Tests/FluentAssertions.Specs/Collections/Data/DataColumnCollectionAssertionExtensionsSpecs.cs

```diff
@@ -0,0 +1,420 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using FluentAssertions.Execution;
+
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs.Collections.Data
+{
+    public static class DataColumnCollectionAssertionExtensionsSpecs
```

Comment:
🔧 I think naming this `DataColumnCollectionAssertionSpecs` is better. That we used extension methods is an implementation detail.

Reply:
Okay :-) I'm slightly of the opposite opinion, that it makes sense to call out the 1-to-1 relationship between the tests and the implementation being tested. Unit tests _are_ tied to implementation by design. But if it feels better to you without the reference to it being implemented with extension methods then I will remove the word. :-)

Reply:
Yes, unit tests are tied to the implementation, but not necessarily to a single class. If we later change the implementation in a way that no longer uses extensions method without breaking the API, the tests should not be affected. We do the same with the `NullubleNumericSpecs`. They all use a functional name.